### PR TITLE
Added defaultExpression() to ColumnSchemaBuilder

### LINKED ISSUE #9445: New migration style (defaultValue('NULL')
As far as I can see in new migration style, there is no ability to define `NULL` default.

* `defaultValue('NULL')` will create `DEFAULT 'NULL'` which is not right
* `defaultValue(null)` will ignore the specified null

### PR DESCRIPTION
**Pull request by chuprik**
Fix for https://github.com/yiisoft/yii2/issues/9337 and... I'm not sure that this is correct but can be used for fix #9445:

``` php
$this->defaultExpression(new Expression('NULL'))
```

### CODE REVIEW

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -105,6 +105,17 @@ public function defaultValue($default)
     }
 
     /**
+     * Specify the default DB Expression for the column.
```
Comment (cebe) [inline framework/db/ColumnSchemaBuilder.php]: better: `Specify the default value for the column in terms of a [[Expression|SQL expression]].`

**Comment by kotchuprik** [comment]
thanks! and sorry for the typo.

**Comment by mikk150** [comment]
what is the status on this one?

**Comment by samdark** [comment]
Merged https://github.com/yiisoft/yii2/commit/e94810551c4ce2c7c5d59dbdcbd46a653ad601bb. Thanks!

**Pull request closed without merging**